src/transformers/quantizers/auto.py

```diff
@@ -129,10 +129,13 @@ def merge_quantization_configs(
         """
         handles situations where both quantization_config from args and quantization_config from model config are present.
         """
-        warning_msg = (
-            "You passed `quantization_config` or equivalent parameters to `from_pretrained` but the model you're loading"
-            " already has a `quantization_config` attribute. The `quantization_config` from the model will be prevail."
-        )
+        if quantization_config_from_args is not None:
+            warning_msg = (
+                "You passed `quantization_config` or equivalent parameters to `from_pretrained` but the model you're loading"
+                " already has a `quantization_config` attribute. The `quantization_config` from the model will be prevail."
```

Comment:
or `will prevail`, but it sounds quite dramatic :) 

```suggestion
                " already has a `quantization_config` attribute. The `quantization_config` from the model will be used."
```

Reply:
hahah yeah, thanks ! 🙏